projects/assets/src/translations/translations.ts

```diff
@@ -5,21 +5,23 @@
  */
 
 import { en } from './en/index';
-import { ja } from './ja/index';
-import { de } from './de/index';
-import { zh } from './zh/index';
+import { extractTranslationChunksConfig } from '@spartacus/core';
```

Comment:
`major:` this line causes the build error described here: https://github.com/SAP/spartacus/pull/19799#issuecomment-2559858954